lib/engine/game/g_18_eus/game.rb

```diff
@@ -153,6 +158,21 @@ class Game < Game::Base
           yellow_20|green_30|brown_40|gray_40
         ].freeze
 
+        def timeline
+          @timeline ||= [
+            'End of OR 1.1: All unsold 2 trains are exported.',
+            'End of OR 1.2: All unsold 2+ trains are exported.',
```

Comment:
Don't know if this is the right place to put it, but start of OR 2.1 should mention going to green.

Reply:
I lifted this timeline from USA, which has the same rule. I don't know if the timeline is only for trains or the whole game. Feel free to add to it if for the whole game.